lib/rex/post/smb/ui/console/command_dispatcher/shares.rb

```diff
@@ -318,7 +344,198 @@ def cmd_cat(*args)
           def cmd_cd_tabs(_str, words)
```

Comment:
I can't comment on it; but can we update `cmd_cat` to use simple client too? The current implementation seems to fail if you cat the same empty file twice

```
SMB (127.0.0.1\foo_share) > cat foo

SMB (127.0.0.1\foo_share) > cat foo
[-] RubySMB::Error::EncryptionError Communication error with the remote host: Socket read returned nil. The server supports encryption but was not able to handle the encrypted request.
```

Whilst download works, which uses the simple wrapper

```
SMB (192.168.123.135\foo) > download foo.txt
[+] Downloaded foo.txt to foo.txt
SMB (192.168.123.135\foo) > download foo.txt
[+] Downloaded foo.txt to foo.txt
```